docs/standard/garbage-collection/fundamentals.md

```diff
@@ -172,32 +173,32 @@ A garbage collection has the following phases:
 
 - A relocating phase that updates the references to the objects that will be compacted.
 
-- A compacting phase that reclaims the space occupied by the dead objects and compacts the surviving objects. The compacting phase moves objects that have survived a garbage collection toward the older end of the segment.
+- A compacting phase that reclaims the space occupied by the dead objects and compacts the surviving objects. The compacting phase moves objects that have survived a garbage collection towards the older end of the segment.
 
-  Because generation 2 collections can occupy multiple segments, objects that are promoted into generation 2 can be moved into an older segment. Both generation 1 and generation 2 survivors can be moved to a different segment, because they are promoted to generation 2.
+  Because generation 2 collections can occupy multiple segments, objects that are promoted into generation 2 can be moved into an older segment. Both generation 1 and generation 2 survivors can be moved to a different segment because they're promoted to generation 2.
 
-  Ordinarily, the large object heap (LOH) is not compacted, because copying large objects imposes a performance penalty. However, in .NET Core and in .NET Framework 4.5.1 and later, you can use the <xref:System.Runtime.GCSettings.LargeObjectHeapCompactionMode%2A?displayProperty=nameWithType> property to compact the large object heap on demand. In addition, the LOH is automatically compacted when a hard limit is set by specifying either:
+  Ordinarily, the large object heap (LOH) isn't compacted because copying large objects imposes a performance penalty. However, in .NET Core and in .NET Framework version 4.5.1 and later, you can use the <xref:System.Runtime.GCSettings.LargeObjectHeapCompactionMode%2A?displayProperty=nameWithType> property to compact the large object heap on demand. In addition, the LOH is automatically compacted when a hard limit is set by specifying either:
```

Comment:
Is there specific guidance to add "version" between .NET Framework and the number?

Reply:
I believe "version" adds more clarity here. However, if it seems redundant/obvious we can remove it. Could you please provide me your suggestion?

Reply:
In this repo, we have 3229 instances without "version" and only 238 instances with "version".

![image](https://user-images.githubusercontent.com/24882762/180469958-8d63f329-54af-492a-9de7-a4dd6b8ef398.png)

![image](https://user-images.githubusercontent.com/24882762/180470044-ce222d9f-cf8f-485a-8337-02220d24955e.png)

Reply:
@gewarren Ok, reverted this change. 🙂👍